src/index.html

```diff
@@ -15,8 +15,37 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="logo"
+        class="logo"
+      />
+      <nav class="nav__list">
+        <ul class="list">
+          <li><a class="is-active nav__link">apple</a></li>
```

Comment:
```suggestion
          <li>
            <a class="is-active nav__link">
              apple
            </a>
          </li>
```

Reply:
The same for other links

Reply:
Text in that links after save looks like this
<li>
      <a class="nav__link">smartphones</a>
</li>, 
maybe prettier makes this formating